tests/org.eclipse.passage.loc.billing.core.tests/src/org/eclipse/passage/loc/billing/core/tests/UserLicensesTest.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Nikifor Fedorov <zelenyhleb@gmail.com> - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.billing.core.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.passage.lic.users.UserLicenseDescriptor;
+import org.eclipse.passage.loc.internal.billing.core.UserLicenses;
+import org.junit.Test;
+
+public class UserLicensesTest {
+
+	@Test
+	public void testGetAllLicenses() {
```

Comment:
Should mention, just in case, that in _JUnit 4_ there is no necessity to start a test method name with `test` prefix - it's enough to be `public`, `void` and `@Test`-annotated.  You can therefor name your test in a less boilerplated manner, like `allLicenses`.